### DISCUSSION TITLE
Best contribution to help support this community

### DISCUSSION THREAD

**Discussion by pablougas** [GrapesJS/grapesjs#5557] [Q&A]
Hi all reading this and @artf, I was part of the great layoff event of 2023 and for a coding challenge I received in an interview, they requested donating 5hrs of work to a favorite open source project and since is-even was all patched up, I thought of my next current favorite @grapesjs. 

I’m a senior software engineer with some handy JS experience and instead of deciding what I think I should contribute, I thought I’d ask what the community or @artf would like help with this holiday season. Even if it’s above the 5hrs, I’ll try and help where I can for the time being of unemployment. 

Ive had a lot of fun playing with grapesjs and in the spirit of giving back to what @artf and others have provided, I hope I can make any open issues or bothersome features go away. Look forward to hearing what you all need support with!

**Comment by artf** [comment]
Hi @pablougas sorry to hear about the layoff but I'm sure soon you'll be able to find your next opportunity 💪

I'm glad to hear you're willing to donate your time to GrapesJS, that means a lot ❤️
Honestly, I don't have anything specific to assign to you especially if you have a constraint on the available time but I can suggest you have a look at the [Roadmap Process](https://github.com/GrapesJS/grapesjs/discussions/5291) I posted a few months back with the relative open [Roadmap](https://github.com/orgs/GrapesJS/projects/1/views/1).

Probably the tasks in the Roadmap are not a great fit for beginners as many of them require a bit of knowledge of the core so I would also suggest these as other important alternatives for contribution.
* **Bug fixes**
  You can check the [Issues list](https://github.com/GrapesJS/grapesjs/issues) for open bugs and if you find anything that you believe you can handle, you're free to pick it up. Feel free to ask any related questions in the issue and if you need any help where to start.
* **Documentation**
  If you had experience working with GrapesJS you might probably faced some struggles with documentation. If you feel there is 
 anything that could be improved, that would be an awesome contribution for the entire community.
* **Component Plugins**
  Creating a component plugin, like a wrapper around some popular JS components (eg. sliders, charts, etc.) it's a great way to get more confidence with the [GrapesJS Component](https://grapesjs.com/docs/modules/Components.html) API and also provide a huge contribution to the community. You can check [Official Plugins](https://github.com/orgs/GrapesJS/repositories?q=-repo%3Agrapesjs%2Fgrapesjs&type=source) and [Community Plugins](https://github.com/topics/grapesjs-plugin) to see a few examples and also understand what would make sense to add.
* **Typescript** 
  If you had experience working with TS and GrapesJS and you found something that could be improved from that point, that's also a welcome contribution.

Wish you all the best
